### DISCUSSION TITLE
copilot work, but can't show suggestinos inline

### DISCUSSION THREAD

**Discussion by ZeyaKim** [community/community#61185] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

![캡처](https://github.com/community/community/assets/118428615/cbabc8a8-bcdf-4dcb-8ef1-77997faa22af)

I started Copilot 1 days ago. but My Copilot can't show inline suggestion. but I can check copilot work but copilot can;t show code suggestion by using other command.

how can I show copilot inline suggestions?

**Comment by Akash1134** [comment, marked as answer]
Hi @ZeyaKim ,

Can you try launching Visual Studio from the Developer Command Prompt using devenv /log? This will make Copilot write extended logging information to the Output > GitHub Copilot pane. 

Also, try the steps below:

*   From the Command Pallette (`Shift` + ⌘ + `P`), select `Check Preferences: Open User Settings
    *   If the value `"editor.inlineSuggest.enabled"` is set to false, set it to `true` and try again.


Let me know how it goes.

Thanks!

**Reply by ZeyaKim** [reply]
my copilot's setting is correct but inline suggestion enabled in editor was false. ty!
